From 8b6b6add69d3f0d66287a3d2b4e951f28e0d7103 Mon Sep 17 00:00:00 2001
From: Michael Natterer <mitch@gimp.org>
Date: Tue, 28 Jan 2014 17:54:46 +0000
Subject: Bug 723177 - After opening an ICC profile with a long description...

...dialogs are too wide

Ellipsize GimpColorProfileComboBox' label.

(cherry picked from commit 219bf3529942f9b12d65cf93705dbeaf0a9812cb)
---
diff --git a/libgimpwidgets/gimpcolorprofilecombobox.c b/libgimpwidgets/gimpcolorprofilecombobox.c
index 10bde3a..0a1b319 100644
--- libgimpwidgets/gimpcolorprofilecombobox.c
+++ libgimpwidgets/gimpcolorprofilecombobox.c
@@ -130,6 +130,11 @@ gimp_color_profile_combo_box_init (GimpColorProfileComboBox *combo_box)
 {
   GtkCellRenderer *cell = gtk_cell_renderer_text_new ();
 
+  g_object_set (cell,
+                "ellipsize", PANGO_ELLIPSIZE_END,
+                NULL);
+
+
   gtk_cell_layout_pack_start (GTK_CELL_LAYOUT (combo_box), cell, TRUE);
   gtk_cell_layout_set_attributes (GTK_CELL_LAYOUT (combo_box), cell,
                                   "text", GIMP_COLOR_PROFILE_STORE_LABEL,
--
cgit v0.9.2
